src/component/checkboxes.html

```diff
@@ -0,0 +1,16 @@
+<template>
+
+  <template containerless repeat.for="option of options">
+    <div class="checkbox">
```

Comment:
Ah. So bootstrap is a soft dependency to `aurelia-orm`. Might be worth documenting.